src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age ? <p className="Person__age">I am {age}</p> : null}
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {partner}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
```

Comment:
Let's do the check inside a p tag so that will have it only once, because now we are repeating the tag and its classname

Reply:
OK. I don`t mind)